WTWSMS/decisions/conversion_decisions.txt

```diff
@@ -533,10 +537,7 @@ targetted_decisions = {
 						NOT = { religion_group = ROOT }
 					}
 				}
-				random = {
-					chance = 60
-					add_trait = sympathy_christendom
-				}
+				add_trait = sympathy_christendom
```

Comment:
The two are mutually exclusive. That would be for a separate event, but for now it should remain random.